CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## 88 Release Notes
 
+### New feature: Prevent Job start during Cloud Quota exhaustion
+
+This optional feature prevents Cromwell from starting new jobs in a billing project that is currently experiencing 
+cloud quota exhaustion. Jobs will be started once the project's quota becomes available. To enable this feature, 
+set `quota-exhaustion-job-start-control.enabled` to true.
+
+Note: Jobs that are being restarted will not be affected by this feature, even if it is enabled.
```

Comment:
I suspect that this note will confuse people, since there are so many kinds of job restarts. I think we can probably just take it out, users probably need to understand a lot about Cromwell internals to even have this question. 😂

Reply:
I see. I added this so that users know that restarts won't affect their jobs not "being restarted" but I didn't know there could be different kind of restarts. I am happy to remove this.